src/security.md

```diff
@@ -39,11 +39,11 @@ See following types of data and the XSS Game below for real-world examples.
 Incoming data from URL parameters is considered the most critical security risk.
 This data might be easily injected from incoming links even if the website is read-only.
 
-Some very simple and unsafe jQuery snippet example:
+Some very simple and unsafe JS snippet example:
 ```
 // Example-URL: `www.domain.com?search=foo`
 // ... grab parameter directly from URL and paste the search word above the results
-$('.search__searchPhrase').html(urlParams['search']);
+document.getElementsByClassName('search__searchPhrase')[0].html(urlParams['search']);
```

Comment:
```suggestion
document.querySelector('search__searchPhrase').innerHTML = urlParams['search'];
```

Why not `querySelector`? And `.html` is a jQuery method (afaik), so use `.innerHTML = ...` instead.

Reply:
Thanks Luke for the Feedback. I've fixed the JS code example and tried to give more context why we're trying to move away from jQuery.